### ISSUE TITLE
Support SM2/SM3 algorithms

### ISSUE THREAD

**Issue by SunFulong** [httpwg/http-extensions#2419]
SM2/SM3 are Chinese cryptographic algorithms: SM3 (hash function) and SM2 (public-private key cryptography), widely used in China, will you support them?

**Comment by martinthomson** [comment]
@SunFulong Does this relate to the message signatures specification or something else?  (SM3 might apply to digest, say.)

**Comment by SunFulong** [comment]
Yes, SM2 is a public key cryptographic algorithm that based on elliptic curves, signature is part of it.

**Comment by martinthomson** [comment]
@SunFulong my question was about which specification you are referring to.  There are multiple specifications in this repository.  I doubt that this applies to [the retrofit work](https://httpwg.org/http-extensions/draft-ietf-httpbis-retrofit.html), for example.

If your goal is to request the addition of SM3 and SM2 to a particular specification, we need to know which.  If this is a more general inquiry, then [our mailing list](mailto:ietf-http-wg@w3.org) is probably a better place to start.
